group_project_v2/mixins.py

```diff
@@ -103,7 +105,7 @@ def user_preferences(self):
         return self._user_preferences(self.project_api, self.user_id)
 
     @staticmethod
-    @memoize_with_expiration(expires_after=timedelta(seconds=5))
+    @memoize_with_expiration(expires_after=DEFAULT_EXPIRATION_TIME)
```

Comment:
I think you should change `memoize_with_expiration` so that `DEFAULT_EXPIRATION_TIME` is the default when no arguments are supplied. You currently use `memoize_with_expiration` nine times, and each use explicitly specifies the same default value.